src/main/java/com/google/api/codegen/DiscoveryFragmentGeneratorApi.java

```diff
@@ -115,6 +126,28 @@ protected void process() throws Exception {
     }
   }
 
+  /*
+   * Parses strings of the format
+   * "go:https://www.hello.com@java:https://www.world.com" and returns the value
+   * of the key corresponding to the key id.
+   */
+  private String parseAuthInstructionsUrl(String authInstructionsUrl, String id) {
```

Comment:
If we did invoke this with multiple languages, then IIUC the code is parsings the serialized map once per language, rather than parsing it once and storing the results in the map. Not a biggie, but not elegant.

Since our current invocation is just one language at a time, it doesn't really matter much right now. But if the code says  we support multiple languages in one invocation, we should make sure it does.

But yeah, totally fine deferring this until we're back on master.